packages/commonwealth/client/scripts/views/pages/discussions/helpers.ts

```diff
@@ -26,9 +27,10 @@ export const isHot = (thread: Thread) => {
 export const isNewThread = (threadCreatedAt: moment.Moment) => {
   const diffInMs = moment().diff(threadCreatedAt);
   return moment.duration(diffInMs).asHours() < 48;
-}
+};
 
 export const getLastUpdate = (thread: Thread): number => {
+  if (!thread) return 0;
```

Comment:
This is a fix discussed in this comment: https://github.com/hicommonwealth/commonwealth/pull/5595#discussion_r1387083555